sdk/keyvault/keyvault-admin/test/public/accessControlClient.spec.ts

```diff
@@ -234,72 +235,74 @@ describe("KeyVaultAccessControlClient", () => {
         roleDefinition.id,
         assertEnvironmentVariable("CLIENT_OBJECT_ID"),
       );
-      assert.equal(assignment.name, assignmentName);
-      assert.equal(assignment.properties?.roleDefinitionId, roleDefinition.id);
-      assert.equal(assignment.properties?.principalId, env.CLIENT_OBJECT_ID);
-      assert.equal(assignment.properties.scope, globalScope);
+      expect(assignment.name).toEqual(assignmentName);
+      expect(assignment.properties?.roleDefinitionId).toEqual(roleDefinition.id);
+      expect(assignment.properties?.principalId).toEqual(env.CLIENT_OBJECT_ID);
+      expect(assignment.properties.scope).toEqual(globalScope);
 
       await client.getRoleAssignment(globalScope, assignmentName);
-      assert.equal(assignment.name, assignmentName);
-      assert.equal(assignment.properties?.roleDefinitionId, roleDefinition.id);
-      assert.equal(assignment.properties?.principalId, env.CLIENT_OBJECT_ID);
-      assert.equal(assignment.properties.scope, globalScope);
+      expect(assignment.name).toEqual(assignmentName);
+      expect(assignment.properties?.roleDefinitionId).toEqual(roleDefinition.id);
+      expect(assignment.properties?.principalId).toEqual(env.CLIENT_OBJECT_ID);
+      expect(assignment.properties.scope).toEqual(globalScope);
 
       await client.deleteRoleAssignment(globalScope, assignmentName);
-      assert.equal(assignment.name, assignmentName);
-      assert.equal(assignment.properties?.roleDefinitionId, roleDefinition.id);
-      assert.equal(assignment.properties?.principalId, env.CLIENT_OBJECT_ID);
+      expect(assignment.name).toEqual(assignmentName);
+      expect(assignment.properties?.roleDefinitionId).toEqual(roleDefinition.id);
+      expect(assignment.properties?.principalId).toEqual(env.CLIENT_OBJECT_ID);
 
-      let error: Error;
       try {
         await client.getRoleAssignment(globalScope, generateFakeUUID());
+        expect.fail("Expected an error to be thrown.");
       } catch (e: any) {
-        error = e;
+        expect(e.message).toMatch(/Requested role assignment not found/);
       }
-      assert.ok(error!.message.match(/Requested role assignment not found/));
     });
 
     it("succeeds when deleting a role assignment that doesn't exist", async () => {
-      await assert.isFulfilled(client.deleteRoleAssignment(globalScope, generateFakeUUID()));
+      await expect(
+        client.deleteRoleAssignment(globalScope, generateFakeUUID()),
+      ).resolves.not.toThrow();
     });
   });
 
-  describe("tracing", () => {
+  // TODO: re-enable as part of migrating to test-utils-vitest https://github.com/Azure/azure-sdk-for-js/pull/30740
```

Comment:
Should be able to use @azure-tools/test-utils-vitest now

Reply:
Nice, that Just Worked ™️ - thanks Matt!

Reply:
Don't forget to remove the TODO comment as well :)